docs/README.md

```diff
@@ -0,0 +1,115 @@
+<!-- Note: As of the writing of this guide all the docs in the docs folder are linked here (excl. subfolders). This is nice but it also means that we have to update this doc every time a new doc is added. Someone should be aware of this.-->
+
+<!--- 
+TODO: 
+Regarding this doc:
+- Proof read
+- Sort the "advanced matters" list at the end of the doc
+
+Regarding docs in general:
+- Add info about ESC calibration
+- Add info about OSD usage
+- Add info about Bluetooth usage
+- Add info about orientation
+- Create a list of all CLI options and meanings
```

Comment:
We actually do have a CLI option reference: https://github.com/cleanflight/cleanflight/blob/master/docs/Cli.md